qvm-tools/qubes-hcl-report

```diff
@@ -69,34 +70,104 @@ while [ $# -gt 0 ]; do
   shift
 done
 
-if [[ "$YAML_ONLY" == 1 ]]
+if [[ "$YAML_ONLY" = 1 ]]
   then
-    if [[ "$SUPPORT_FILES" == 1 ]]
+    if [[ "$SUPPORT_FILES" = 1 ]]
       then
         echo -e "ERROR: --yaml-only is mutually exclusive with --support"
         exit 1
     fi
-    if [[ "$1" ]]
+    if [[ -n "$1" ]]
       then
         echo -e "ERROR: --yaml-only is mutually exclusive with providing a VM name"
         exit 1
     fi
 fi
 
 
-DATE=`date +%Y%m%d-%H%M%S`
-
-
-TEMP_DIR=`mktemp --tmpdir -d HCL.XXXXXXXXXX`
-cat /etc/qubes-release > $TEMP_DIR/qubes-release
```

Comment:
All used files should still be copied into temp dir, otherwise the support archive will be incomplete.